src/pages/Cuisine.js

```diff
@@ -1,19 +1,25 @@
 import React, { useState, useEffect } from "react";
-import { motion } from "framer-motion";
-import { Link, useParams } from "react-router-dom";
 import "./Cuisine.css";
-import { API_KEY, API_URL } from "../Routes";
+import { API_URL } from "../Routes";
+import { Link, useParams } from "react-router-dom";
 
 function Cuisine() {
   const [cuisine, setCuisine] = useState([]);
   let params = useParams();
 
   const getCuisine = async (name) => {
-    const data = await fetch(
-      `${API_URL}complexSearch?apiKey=${API_KEY}&cuisine=${name}`
-    );
-    const recipes = await data.json();
-    setCuisine(recipes.results);
+    try {
+      const data = await fetch(
+        `${API_URL}complexSearch?apiKey=${process.env.REACT_APP_API_KEY}&cuisine=${name}`
+      );
+      const recipes = await data.json();
+      if (recipes.status === "failure") {
```

Comment:
właśnie podobny warunek powinien być we wszystkich miejscach gdzie odpytujemy API